rollup.config.ts

```diff
@@ -156,6 +156,16 @@ export default function rollup (options: RollupOptions): RollupOptions[] {
         vue: 'Vue',
         '@w3ui/vue-keyring': 'VueKeyring'
       }
+    }),
```

Comment:
do we need rollup to build this? afaik the rollup output is only used in built packages, and I'm not sure we want this particular package to be published. could be wrong, just curious what needs this...